### ISSUE TITLE
new dashboard continued

### ISSUE THREAD

**Issue by specu** [sealemar/BingRewards#338]
so there's stuff in the dropdown menu that's not on the dashboard
https://www.bing.com/rewardsapp/bepflyoutpage?style=modular
in addition to that, 3 of tasks in the dropdown menu about are clickable but they dont earn any points.
Regardless, perhaps it would be worthwhile to parse both the dashboard and the dropdown menu from the link above?

**Comment by baltimoron** [comment]
Good idea but I think the #1 priority at this point is figuring out why the hits won't credit. I'm really stumped on that one.

**Comment by specu** [comment]
I think hits would credit via this flyout page. it might be the case that the old parser is just going to work with this page out of the box.

**Comment by specu** [comment]
Apparently the rewards that show at that dashboard link that are clickable but dont earn points show up on the main dashboard as tomorrow's activities.

**Comment by baltimoron** [comment]
@specu have you seen any instances of hits on the new dashboard crediting when executed by the script?

**Comment by specu** [comment]
@baltimoron my impression was that no. even when you click manually it seems delayed like it uses ajax to check if it was executed. I'll monitor and report back today.

**Comment by baltimoron** [comment]
Yeah when you log in and manually click on a reward there's definitely a delay before it credits. I'm not sure what the exact mechanism is though.

**Comment by zengfu94** [comment]
@baltimoron chrome network tab, u didnt hear it from me :)
telegram: @rocketmanben if you need some more help

**Comment by baltimoron** [comment]
@zengfu94 yeah I've been investigating with the network tab for a while now but nothing stands out to me. I think it's a little over my head.

**Comment by zengfu94** [comment]
@baltimoron make sure to preserve log, search for reportActivity and you'll find what you need
basically the workflow I use
network tab preserve log > perform action > check tab > copy as curl (bash) > curl.trillworks.com to convert to python requests

**Comment by baltimoron** [comment]
@zengfu94 this is awesome, thanks.

**Comment by baltimoron** [comment]
@zengfu94 did you convert the script over to the requests library? Not sure how to replicate that using the urllib2 syntax.

**Comment by baltimoron** [comment]
sheeeeeeeeiiiiiit @zengfu94 you are the man.

**Comment by specu** [comment]
@baltimoron do you feel like you're getting closer to getting hits to work?

**Comment by baltimoron** [comment]
Got it working, just need to do some cleanup before I commit it.

**Comment by specu** [comment]
I realize this is premature but I'm hearing some good things about streaks. I haven't looked into that myself but it would be great to get any extra points that are available.

**Comment by baltimoron** [comment]
Yeah except quizzes can also be required for streaks so we would need a way to handle those too.

**Comment by specu** [comment]
it seems like module requests is not included in all the distributions(?)

**Comment by specu** [comment]
I'm also seeing
Points before:         0
Points after:          0
Points earned:         0
with this latest commit (after first run)

**Comment by baltimoron** [comment]
I just ran with a fresh pull of my fork and the hits all credited.

**Comment by specu** [comment]
ran fine on raspbian. I got that odd result on slackware.

**Comment by specu** [comment]
looks like it might have been a 1 time glitch

**Comment by specu** [comment]
that accounts was actually stuck on "confirm your security info". works now.

**Comment by specu** [comment]
I got the following "error" today:
name        : Candelabra guy
   Error    :   true
   Message  : Expected to earn 10 points, but earned 55 points. Check error_20180204-043205.469630.html for further information
I guess the streak kicked in.

**Comment by zengfu94** [comment]
@baltimoron haha congrats, pm on tele if u need anymore help

**Comment by specu** [comment]
@baltimoron i think your fork is now a bit behind the main tree. (my #343 and #346 got accepted but there might be more.
I'm looking at quizzes the same way as above - looking at network traffic and then translating to pything in curl.trillworks.com

**Comment by baltimoron** [comment]
@specu if you want another set of eyes on the quiz stuff let me know and I should be able to take a look on Friday.

**Comment by specu** [comment]
@baltimoron i've put what I found in #351  ( @zengfu94  )
Selfishly with nice rewards this month, I'm trying to keep those streaks alive, manually if needed and possible. Although they break them sometimes even if all activities completed. I dont know if those streaks are economic either. it seems like for 3 days you get 45 points and for 7 days you get 75 points. so for 3 day streak, 1 day off and another 3 day streak you get 70 points, almost no difference. Anyways I think it would make sense to report streaks (as an activity in the report for example) just to report if all the activities needed for the streak have been completed.
